tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -44,17 +44,14 @@ trait Sharing {
 	 *
 	 * @var SimpleXMLElement[]
 	 */
-	private array $lastShareDataByUser = [];
+	private ?SimpleXMLElement $userSharesData = null;
 
 	/**
-	 * Contains the share id of the last share that was created by each user,
-	 * either using the Sharing API or on the web UI.
+	 * Contains all the user share data
 	 *
-	 * @var string[]
+	 * @var array
 	 */
-	private array $lastShareIdByUser = [];
-
-	private ?string $userWhoCreatedLastShare = null;
+	private array $createdUserShares = [];
```

Comment:
If it's storing share ids
```suggestion
	private array $createdUserShareIds = [];
```

Reply:
or planning to store other items also?

Reply:
It is supposed to store other items as well, but currently only share id is being stored.